### ISSUE TITLE
"Quick mode" for tags

### ISSUE THREAD

**Issue by jamesdunham** [pelagios/recogito2#376]
I've been playing with recogito2 on a local deployment for a couple of days and think it could be incredibly useful. Thanks for all your work.

I'd be interested in using tags in "quick mode," which IIUC was one of the unimplemented UI features referenced in #40. Is this still planned? 

My goal is actually to annotate text for entities other than people and places. So defining custom annotation types would of course be ideal. But if using tags + types together instead, having "quick mode" for tags would greatly reduce friction.

**Comment by rsimon** [comment]
Yes, that is indeed still planned. The idea is that the **Tags** option in the **Quick** dropdown would let you define one (or more) tags; and then you could just go through the text and apply those tags quickly, without the editor box popping up. Just in the same way as you can now do for places and persons.

Unfortunately, I can't give an exact time for when we'll be able to add this. But it's definitely coming.

**Comment by rsimon** [comment]
I'd say we leave it open, since there's no dedicated issue for this anyway. And this way we'll keep in on the radar ;-)
